ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.18.yaml

```diff
@@ -228,6 +228,7 @@ tests:
       AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
       BASE_DOMAIN: qe.devcluster.openshift.com
       EDGE_ZONE_TYPES: wavelength-zone
+      ENABLE_OTA_TEST: OCP-73352
```

Comment:
the same https://github.com/openshift/release/pull/60896#discussion_r1929917064

Reply:
I was thinking this new command is introduced from 4.18, so we don't need to test it in 4.17 -> 4.18 jobs.

Reply:
not really, as i mentioned in https://github.com/openshift/release/pull/60896#discussion_r1929937076, 4.19 oc will be tested in 4.18-4.19 while 4.18 will be in 4.17-4.18.

Reply:
done